steno/src/model/audio_suppresser/audio_suppresser.cpp

```diff
@@ -3,40 +3,39 @@
 namespace Model
 {
 /**
- * @brief Take audio buffer in input and remove sources in sourcesToSuppress/
- * @param [IN] sourcesToSuppress - vector of audio sources to remove.
+ * @brief Removes all audio sources that are not in the vector sourcesToKeep
+ * @param [IN] sourcesToKeep - vector of audio sources to keep.
  * @param [IN/OUT] audioBuf - Audio buffer to modify
  * @param [IN] bufferLength
  */
-void AudioSuppresser::suppressSources(const std::vector<int> &sourcesToSuppress, uint8_t *audioBuf,
-                                      const int bufferLength)
+void AudioSuppresser::suppressNoise(const std::vector<int> &sourcesToKeep, uint8_t *audioBuf, const int bufferLength)
```

Comment:
I would pass an audio chunk to this function instead of the raw buffers.
I know this is not your doing, but I would put the channels number and bytes per channel in the audio chunk, so you can use them here.